src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -25,6 +25,10 @@ public CommandResult(String feedbackToUser, List<? extends ReadOnlyPerson> relev
         this.feedbackToUser = feedbackToUser;
         this.relevantPersons = relevantPersons;
     }
+    
+    public String getFeedBack() {
```

Comment:
would be good to add the method header comment to this new getter. as mentioned in class, "Feedback" is one word, so there is no need for capital case 'b' in the name.